pkgs/applications/blockchains/trezor-suite/default.nix

```diff
@@ -53,6 +53,10 @@ appimageTools.wrapType2 rec {
   '';
 
   meta = with lib; {
+    # this application does not perform its core functionality.
+    # ref: https://github.com/NixOS/nixpkgs/issues/281975
+    broken = true;
```

Comment:
I saw the discussion in #281975. Is the package only broken for you or is it also the case for @prusnak, the package maintainer?

Reply:
yes, i see the same issue, but have no time to fix this now